acm/ivy.xml

```diff
@@ -33,7 +33,10 @@
     <dependency org="com.seaglasslookandfeel" name="seaglasslookandfeel" rev="0.2.0-JAVA8"/>
     <dependency org="org.swinglabs" name="swingx" rev="1.6.1"/>
     <dependency org="args4j" name="args4j" rev="2.0.16"/>
-    <dependency org="commons-io" name="commons-io" rev="2.4"></dependency>
-    <dependency org="net.sf.opencsv" name="opencsv" rev="2.3"></dependency>
+    <dependency org="commons-io" name="commons-io" rev="2.4"/>
+    <dependency org="net.sf.opencsv" name="opencsv" rev="2.3"/>
+    <dependency org="org.powermock" name="powermock-api-mockito" rev="1.6.4" />
```

Comment:
Do you think it's possible to force ivy to include these powermock dependencies only during the test target? Keeping the acm deploy jar file small in size is preferable due to slow internet connections in Ghana.